### PR TITLE
Backport #2428 to v0.17

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: src/render/Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 [package]
 name = "gfx"
-version = "0.17.1"
+version = "0.17.2"
```
Comment (kvark) [inline src/render/Cargo.toml]: we shouldn't touch this version, let's instead touch the `src/core` version

Comment (torkleyy) [reply src/render/Cargo.toml]: core version is already bumped

Comment (torkleyy) [reply src/render/Cargo.toml]: Done.

**Pull request merged**